cw_infrastructure/cw_infrastructure/clearwater_options.py

```diff
@@ -29,7 +23,7 @@ def get_options():
         # namespace is to be used
         sas_server_validator = vlds.resolvable_ip_or_domain_name_validator
 
-        if ClearwaterOptions.get_value('sas_use_signaling_interface') == 'Y':
+        if utils.get_environment_variable('sas_use_signaling_interface') == 'Y':
```

Comment:
I don't think the schema gets to decide where the options come from - this is the job of the infrastructure. The infrastructure should expose a method to allow the schema to access options, but I don't think this should be `get_option` or something rather than `get_environment_variable`.